target-platform/target-platform.target

```diff
@@ -82,7 +82,7 @@
 				<dependency>
 					<groupId>com.google.code.gson</groupId>
 					<artifactId>gson</artifactId>
-					<version>2.9.1</version>
+					<version>2.10.1</version>
```

Comment:
Please keep the dependency as it is. Due to lsp4j's package imports, that causes TP resolution errors in the workspace:
https://github.com/eclipse-m2e/m2e-core/pull/1146#issuecomment-1362187746

I don't know why it doesn't do that in the CI.

Reply:
I don't think this issue is an issue of m2e and that the resolution error in the TP is something m2e should be blamed for. There is nothing that prevents gson 2.9.1 and 2.10.1 to coexist in the same TP. If LSP4J requires 2.9.1, then consumers of LSP4J must provide 2.9.1; without preventing m2e to move to newer version.

Reply:
Then please add both versions, because only with 2.10 I can currently not launch an IDE in my m2e workspace.
The update for lsp4j happens in https://github.com/eclipse/lsp4j/pull/689.

Reply:
I believe both versions are supposed to be part of the resolved TP: 2.10.1 explicitly changed here, and 2.9.1 as a transitive dependency of LSP4E. Can you check your resolved TP contains both? Then can you check your launch config contains both?

Reply:
> I believe both versions are supposed to be part of the resolved TP: 2.10.1 explicitly changed here, and 2.9.1 as a transitive dependency of LSP4E. Can you check your resolved TP contains both? Then can you check your launch config contains both?

Indeed the TP contains both. Nevertheless PDE seems to fail to wire the lsp4j to the old gson-bundle. gson isn't a singleton, so this look like a bug in PDE to me.